src/logger.js

```diff
@@ -4,8 +4,15 @@ const logger = createLogger({
   exitOnError: false,
   level: process.env.LOG_LEVEL || 'debug',
   format: format.combine(
+    format.colorize({ all: true }),
```

Comment:
I was gonna do this yesterday. Big fan

Reply:
Like this?
![](http://i1.wp.com/eveningharold.com/wp-content/uploads/2018/03/massive-fan.jpg)

Reply:
😂 exactly